Makefile

```diff
@@ -55,6 +55,10 @@ arm64:
 s390x:
 	GOOS=linux GOARCH=s390x go build -mod=vendor $(LDFLAGS) -o bin/tkn-linux-s390x ./cmd/tkn
 
+.PHONY: ppc64le
+ppc64le:
+	GOOS=linux GOARCH=ppc64le go build -mod=vendor $(LDFLAGS) -o bin/tkn-linux-s390x ./cmd/tkn
```

Comment:
Copy-paste error, you've got a stray s390x at the end of that.

Reply:
indeed :sweat_smile: